mgmt/rest/server.go

```diff
@@ -492,6 +507,41 @@ func (s *Server) addRoutes() {
 		s.r.GET("/v1/tribe/members", s.getMembers)
 		s.r.GET("/v1/tribe/member/:name", s.getMember)
 	}
+
+	// profiling tools routes
+	if s.pprof {
```

Comment:
> are you sure that you want to dynamically change it when snapd is already running? 

We don't enable/disable profiling tools without re-starting Snap. We do the same way than tribe. Else we would need another route to the API, in order to handle dynamic config changes. And I'm not sure we want that.